<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Offset Angle</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QDoubleSpinBox" name="Offset_Angle">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <double>-360.000000000000000</double>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_15">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Algorithm</string>
      </property>
     </widget>
    </item>
    <item row="23" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="14" column="1">
     <widget class="QComboBox" name="algorithm_list">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>gridrec</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>art</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bart</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>fbp</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>sirt</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tv</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>grad</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mlem</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tikh</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>osem</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>pml_hybrid</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>pml_quad</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ospml_hybrid</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ospml_quad</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QComboBox" name="filter_list">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>shepp</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ramlak</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>hann</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>hamming</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>parzen</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>butterworth</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cosine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>none</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDoubleSpinBox" name="COR">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QDoubleSpinBox" name="speed_W">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
      <property name="value">
       <double>0.300000000000000</double>
      </property>
     </widget>
    </item>
    <item row="18" column="0" colspan="2">
     <widget class="QPushButton" name="pushReconstruct_all">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Reconstruct
 Complete Volume</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QDoubleSpinBox" name="speed_Y">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>brightness</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_32bit_float">
        <property name="text">
         <string>32-bit float</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_16bit_integer">
        <property name="text">
         <string>16-bit integer scaling</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>slice</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QPushButton" name="pushReconstruct">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>COR:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QSpinBox" name="brightness">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>640000</number>
      </property>
      <property name="value">
       <number>32000</number>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Y-speed[Px/img]</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Rot-speed[Â°/img]</string>
      </property>
     </widget>
    </item>
    <item row="20" column="0">
     <widget class="QDoubleSpinBox" name="int_low">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-65535.000000000000000</double>
      </property>
      <property name="maximum">
       <double>65535.000000000000000</double>
      </property>
      <property name="value">
       <double>-10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="20" column="1">
     <widget class="QDoubleSpinBox" name="int_high">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-65535.000000000000000</double>
      </property>
      <property name="maximum">
       <double>65535.000000000000000</double>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="2" rowspan="22">
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>2022</width>
         <height>2022</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="1">
         <widget class="QLabel" name="test_reco">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>2000</width>
            <height>2000</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>5000</width>
            <height>5000</height>
           </size>
          </property>
          <property name="text">
           <string>test_reco</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="slice_number">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>5000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0" rowspan="2" colspan="2">
     <widget class="QPushButton" name="pushLoad">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_16">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
    </item>
    <item row="21" column="1">
     <widget class="QTextBrowser" name="max">
      <property name="maximumSize">
       <size>
        <width>119</width>
        <height>23</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="21" column="0">
     <widget class="QTextBrowser" name="min">
      <property name="maximumSize">
       <size>
        <width>119</width>
        <height>23</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
